docs/UserGuide.md

```diff
@@ -125,25 +183,25 @@ Format: `s-update A/NUM B/NUM C/NUM ...`
 Example:
 * `s-update BSBM/100 BSBBT/120` Updates the sales of Brown Sugar Boba Milk `BSBM` to 100 and
  Brown Sugar Boba Black Tea `BSBBT` to 120.
- 
+
  <div markdown="span" class="alert alert-primary">
- 
+
 :information_source: Note about the update: <br>
 
 The record in tCheck will be overwritten by the input that you provide. <br>
- 
+
  </div>
 
-#### 1.2 Listing the number of drinks sold : `s-list`
-Shows a list of all types of drinks sold in the Sales Tracker.
+#### 1.2 Listing the number of drinks sold for the day : `s-list`
+Shows a list of all types of drinks sold for the current day.
```

Comment:
For line 195 & 196. A similar problem here, you may want to change back to what it was. Thanks